apps/health_alerts/models.py

```diff
@@ -0,0 +1,89 @@
+import datetime
+
+from django.db import models
+
+from wagtail.admin.edit_handlers import FieldPanel
+from wagtail.core.models import Page
+from wagtail.documents.edit_handlers import DocumentChooserPanel
+
+
+class HealthAlertIndexPage(Page):
+    # There can be only one HealthAlertIndexPage
+    max_count = 1
+    # ... and its children must be HealthAlertPages
+    subpage_types = ["health_alerts.HealthAlertPage"]
+
+    def get_context(self, request):
+        """
+        Add health_alerts queryset and right_nav_headings to context.
+        """
+        context = super().get_context(request)
+
+        # Get all live HealthAlerts, ordered date descending.
+        health_alerts = (
+            HealthAlertPage.objects.child_of(self).order_by("-alert_date").live()
+        )
+        context["health_alerts"] = health_alerts
+
+        # Get list of each year that we have an alert for. This is used for grouping of
+        # alerts on the page, as well as to trigger the right scroll bar to be created.
+        years = [alert.alert_date.year for alert in health_alerts]
+        # the following line removes duplicates but keeps things ordered (unlike sets)
+        years = list(dict.fromkeys(years))
+        context["right_nav_headings"] = years
+        return context
+
+
+class HealthAlertPage(Page):
+    parent_page_types = ["health_alerts.HealthAlertIndexPage"]
+    alert_file = models.ForeignKey(
+        "wagtaildocs.Document", null=True, blank=True, on_delete=models.SET_NULL
+    )
+
+    class Priority(models.IntegerChoices):
+        UPDATE = 1
+        NOTIFICATION = 2
+        ADVISORY = 3
+        ALERT = 4
+
+    priority = models.IntegerField(choices=Priority.choices)
+
+    alert_date = models.DateField(default=datetime.date.today)
```

Comment:
This field currently is being displayed as `yyyy/mm/dd`. To be consistent with `Contact Us` snippet and other areas we are displaying a date in the admin. Should we use `mm/dd/yyyy`?